api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirObservationServiceImpl.java

```diff
@@ -43,7 +44,7 @@
 	private FhirObservationDao dao;
 	
 	@Autowired
-	private ObservationTranslator observationTranslator;
+	private Translator translator;
```

Comment:
This should be `private ObservationTranslator translator`;

Reply:
What's happening here is that we're using Lombok to automatically generate a getter and setter for the field. By naming the field `translator`, Lombok automatically generates a `getTranslator()` function, which the `BaseFhirService` class requires.

Reply:
Finished

Reply:
Actually i was asking my self why we change to translator but hanks